rodan_job/base.py

```diff
@@ -527,7 +527,7 @@ def run_my_task(self, inputs, settings, outputs):
 
         # Automatic grouping and reclassifying
         elif settings['@state'] == ClassifierStateEnum.GROUP_AND_CLASSIFY:
-            # CLASSIFYING STAGE
+            # GROUP AND CLASSIFY STAGE
```

Comment:
Correct. If we are grouping and classifying we should ask the user how to perform the grouping. 

I think you can get some ideas about what these values mean by checking https://github.com/DDMAL/Interactive-Classifier/wiki/How-to-Use#grouping-function or, better, by reading the Gamera code.